chromebook-setup.sh

```diff
@@ -410,7 +452,9 @@ cmd_format_storage()
     wait_for_partitions_to_appear
     find_partitions_by_id
 
+    sudo mkfs.ext4 -L kernel "$CB_SETUP_STORAGE1"
     sudo mkfs.ext4 -L ROOT-A "$CB_SETUP_STORAGE2"
+    sudo cgpt add -i 1 -T 5 -P 10 -t kernel -l kernel "$CB_SETUP_STORAGE"
```

Comment:
This should not be needed. Is not working if you remove this line and the above that you added?